project4/styles.css

```
@@ -0,0 +1,490 @@
+    --color-offwhite: #f7f7f7;
+}
+
+/*  ================== COLOR PALETTE ==================*/
+
+*,
+*::before,
+*::after {
+    box-sizing: border-box;
+}
+
+body {
+    background-color: var(--color-offblack);
+    background-image: url("https://www.transparenttextures.com/patterns/low-contrast-linen.png");
+    color: var(--color-offwhite);
+    font-family: 'Maven Pro', sans-serif;
+    font-size: 16px;
+    line-height: 25px;
+    padding: 0 5%;
+}
+
+footer {
+    background-color: inherit;
+    background-image: inherit;
+    opacity: 60%;
+    color: inherit;
+    font-family: inherit;
+    font-size: inherit;
+    line-height: inherit;
+}
+
+h1 {
+    font-size: 60px;
+    height: 20px;
+    line-height: 60px;
+    text-transform: uppercase;
+    padding-bottom: 20px;
+}
+
+h1:hover {
+    color: var(--color-lightgreen);
+}
+
+h2 {
+    font-size: 34px;
+    font-weight: 500;
+    padding-top: 20px;
+    border-top: 1px solid var(--color-lightgreen);
+}
+
+h3 {
+    font-size: 22px;
+    margin-top: 0;
+    margin-bottom: 10px;
+    padding-bottom: 5px;
+}
+
+h3:hover {
+    color: var(--color-darkgreen);
+}
+
+h4 {
+    font-size: 24px;
+    text-transform: uppercase;
+    margin-bottom: 10px;
+    margin-top: 50px;
+}
+
+input {
+    font-family: inherit;
+    font-size: 18px;
+    resize: horizontal;
+    resize: vertical;
+    height: 40px;
+    width: 320px;
+}
+
+li {
+    font-size: 12px;
+    display: inline-block;
+    margin-top: 10px;
+    text-transform: uppercase;
+}
+
+li+li {
+    margin-left: 49px;
+}
+
+li a {
+    color: var(--color-offwhite);
+    text-decoration: none;
+    padding: 2px;
+    border-bottom: 1.5px solid var(--color-offwhite);
+}
+
+li a:hover {
+    color: var(--color-darkgreen);
+    border-bottom-color: transparent;
+}
+
+nav {
+    -webkit-filter: drop-shadow(4px 4px 4px black);
+    filter: drop-shadow(4px 4px 4px black);
+    background-color: var(--color-offblack);
+    background-image: url("https://www.transparenttextures.com/patterns/low-contrast-linen.png");
+    height: 90px;
+    left: 0;
+    width: 100%;
+    position: fixed;
+    top: 0;
+    z-index: 10;
+}
+
+span {
+    font-size: 24px;
+    font-weight: 900;
+}
+
+ul {
+    display: inline-block;
+    list-style: none;
+    text-align: center;
+    width: 100%;
+    margin: 0;
+    padding-left: 0;
+    position: relative;
+    top: 10px;
+}
+
+/*  ================== GLOBAL SETTINGS ================== */
+
+.button {
+    background: var(--color-offwhite);
+    border: 5px solid var(--color-offwhite);
+    color: var(--color-offblack);
+    cursor: pointer;
+    display: block;
+    font-family: inherit;
+    font-size: 20px;
+    font-weight: 700;
+    height: 70px;
+    margin: auto;
+    width: 200px;
+    padding-top: 15px;
+    text-align: center;
+    text-decoration: none;
+    text-transform: uppercase;
+}
+
+.button:hover {
+    background: var(--color-lightgreen);
+    border: 5px solid var(--color-lightgreen);
```

I recommend only writing styles for what _changes_ on hover (or focus or any other state change or a media query) -- so, here, the `border` shorthand is making you avoid that; the `border-width` and `border-style` are all the same here as on line 138. So, it would be better to update this line to
```suggestion
    border-color: var(--color-lightgreen);
```